packages/server/graphql/queries/helpers/fetchAllRepoIntegrations.ts

```diff
@@ -0,0 +1,23 @@
+import {GQLContext} from '../../graphql'
+import {GraphQLResolveInfo} from 'graphql'
+import fetchAtlassianProjects from './fetchAtlassianProjects'
+import fetchGitHubRepos from './fetchGitHubRepos'
+
+const fetchAllRepoIntegrations = async (
+  teamId: string,
+  userId: string,
+  context: GQLContext,
+  info: GraphQLResolveInfo
+) => {
+  const {dataLoader} = context
+  const [jiraProjects, githubRepos] = await Promise.all([
+    fetchAtlassianProjects(teamId, userId, context),
+    fetchGitHubRepos(teamId, userId, dataLoader, context, info)
+  ])
+  const getValue = (item) => (item.nameWithOwner || item.name).toLowerCase()
+  return [...jiraProjects, ...githubRepos].sort((a, b) => {
```

Comment:
-1 we're doing half a sort here, not sorting in `allAvailableRepoIntegrations`, and sorting in `repoIntegrations`.
I propose we do all the sorting here

Reply:
That makes sense! I've pushed an update with that change